ToDoApp/Backend/src/resolvers/todo/todos.spec.js

```diff
@@ -1,35 +1,75 @@
-const { mergeResolvers } = require("merge-graphql-schemas");
-const { createTestClient } = require('apollo-server-testing');
-const { ApolloServer, gql } = require('apollo-server');
-const { typeDefs } = require("../../schema/typeDefs");
-const { userResolver } = require("../user/userResolver");
-const { todoResolver } = require("./todoResolver");
+const {
+    mergeResolvers
+} = require("merge-graphql-schemas");
+const {
+    createTestClient
+} = require('apollo-server-testing');
+const {
+    ApolloServer,
+    gql,
+    makeExecutableSchema
+} = require('apollo-server');
+const {
+    typeDefs
+} = require("../../schema/typeDefs");
+const {
+    userResolver
+} = require("../user/userResolver");
+const {
+    todoResolver
+} = require("./todoResolver");
+const {
+    augmentSchema
+} = require("neo4j-graphql-js");
+const neo4j = require('neo4j-driver');
 
+const driver = neo4j.driver(
+    'bolt://localhost',
+    neo4j.auth.basic('neo4j', 'password'), {
```

Comment:
Parameterize this please and put that outside of the source code. Think of people who don't have this set of credentials.